tests/test_kci.py

```diff
@@ -27,9 +30,9 @@ def test_given_continuous_dependent_data_when_perform_kernel_based_test_then_rej
 
          X <- Z -> Y
     """
-    z = np.random.randn(1000, 1)
-    x = np.exp(z + np.random.rand(1000, 1))
-    y = np.exp(z + np.random.rand(1000, 1))
+    z = rng.standard_normal((1000, 1))
+    x = np.exp(z + rng.uniform(size=(1000, 1)))
+    y = np.exp(z + rng.uniform(size=(1000, 1)))
```

Comment:
Why uniform instead of Gaussian noise? It doesn't matter here, just curious to hear.

Reply:
`rand` is `uniform` essentially. I think this is just an example I followed off the literature. I think it also helps limit the amount of variability since there's a nonlinear transformation going on.

I just tried with `standard_normal` and it works too actually. Preference on keeping either one?

Reply:
`rand` is uniform :p: https://numpy.org/doc/stable/reference/random/generated/numpy.random.rand.html

Okay yeah it seems robust, so I'll just leave it as it was previously.

Reply:
Oh there is `rand` and `randn` haha
Seems that was even used in a mix before.